CHANGELOG.md

```diff
@@ -7,14 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
-* Update template to tools `1.10.2`
```

Comment:
Why did you remove this line @yuukiiwa ?

Reply:
I am sorry, @drpatelh! I did that accidentally...

Reply:
No worries :) `dev` container has been built and pushed to Dockerhub now so you can re-pull it and it should work.

Reply:
Thanks! Will do that soon!